macos/FluentUI/Button/Button.swift

```diff
@@ -607,3 +613,5 @@ fileprivate struct ButtonSizeParameters {
 		}
 	}
 }
+
+fileprivate let OnePointSize = NSSize(width: 1, height: 1)
```

Comment:
Can you use private instead of fileprivate? swiftlint will complain about it with a warning.
Or even better: just create it directly in the scope that you're using it.

Reply:
We should probably update [aka.ms/swiftguide](https://microsoft.github.io/swift-guide/FileContentOrdering.html) to reflect we don't like fileprivate anymore

Reply:
Annnnnnd PR sent https://github.com/microsoft/swift-guide/pull/9

Reply:
And I figured that the mac folder does not have swiftlint validation:
#390

Reply:
Yeah, I was supposed to add that a while back for an FHLand I think I got hung up on sharing the yml / choosing SwfitLint over SwiftFormat. Thanks for adding it now!

Reply:
Moved this line into a narrower scope and removed all other 'fileprivate' usage in Button.swift.